Fixes
cc1: error: invalid parameter `max-inline-insns'
make[2]: *** [/home/dank/wk/crosstool-0.28-rc6/build/powerpc64-linux-gnu/gcc-3.4.0-glibc-2.3.2/build-glibc/elf/dl-load.o] Error 1
when building with gcc-3.4.0, which no longer has a single
--max-inline-insns parameter.

http://sources.redhat.com/cgi-bin/cvsweb.cgi/libc/sysdeps/powerpc/powerpc64/Makefile.diff?r1=1.3&r2=1.4&cvsroot=glibc

===================================================================
RCS file: /cvs/glibc/libc/sysdeps/powerpc/powerpc64/Makefile,v
retrieving revision 1.3
retrieving revision 1.4
diff -u -r1.3 -r1.4
--- libc/sysdeps/powerpc/powerpc64/Makefile     2003/10/22 21:30:03     1.3
+++ libc/sysdeps/powerpc/powerpc64/Makefile     2004/02/20 05:43:52     1.4
@@ -20,5 +20,5 @@
 
 ifeq ($(subdir),elf)
 # help gcc inline asm code from dl-machine.h
-+cflags += --param max-inline-insns=2000
++cflags += -finline-limit=2000
 endif

